Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -2292,48 +2289,12 @@ public void RemovePlugin(IPlugin plugin)
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <param name="progress">The progress.</param>
         /// <returns>Task{CheckForUpdateResult}.</returns>
-        public async Task<CheckForUpdateResult> CheckForApplicationUpdate(CancellationToken cancellationToken, IProgress<double> progress)
+        public Task<CheckForUpdateResult> CheckForApplicationUpdate(CancellationToken cancellationToken, IProgress<double> progress)
```

Comment:
Nuke this instead of stubbing.

Reply:
It is needed by the Interface it implements, it doesn't do any harm and makes it easier to re-implement.

Reply:
It's useless code until then and may not be the way we reimplement it.

Can the interface be changed instead?